### ISSUE TITLE
Can't start language server in neovim with executable from 1.2.3 release

### ISSUE THREAD

**Issue by alextes** [jaredly/reason-language-server#161]
Some versions that might be handy:
neovim: 0.3.1
languageclient-neovim: 283dba5
vim-reason-plus: e446079
reason-language-server: 1.2.3
OS: macOS Mojave

Steps I followed were: 
Install the language server plugin for neovim. 
Install the `vim-reason-plus` plugin.
Grab the latest release for macOS.
Configure the path for the language server in vim.
Open a reason file.
execute `:LanguageClientStart`
Output in vim:
```
LanguageClient project root: /Users/alextes/code/rtl-reason/src
[Error] Invalid_argument("String.sub / Bytes.sub")Raised at file "pervasives.ml", line 33, characters 20-45^@Called from file "string.ml" (inlined), line 47, characters 2-23^@Called from file "util/MerlinFile.re", line 107, characters 23-67^@Called from file "src/analyze/BuildSystem.re", line 108, characters 28-59^
@Called from file "util/Monads.re", line 203, characters 18-24^@Called from file "src/analyze/State.re", line 117, characters 24-60^@Called from file "src/analyze/State.re", line 493, characters 24-73^@Called from file "src/lsp/NotificationHandlers.re", line 75, characters 24-72^@Called from file "src/lsp/BasicServ
er.re", line 84, characters 11-35
Language server reason exited unexpectedly: failed to fill whole buffer
```

VSCode works fine with the extension. Interestingly neovim worked fine too before but this morning I noticed that the `reason-language-server` binary had disappeared from my global node binaries (not sure which package put it there, maybe `bs-platform` 🤔 ?), and replaced it with the executable from the release.

Let me do if there's something I can do to clarify matters!

Thanks for an awesome language server 🙏 !

**Comment by j-m-hoffmann** [comment]
same here neovim 0.3.1 rls 1.2.3 on linux, rls 1.1.1 works fine.

HUGE Thank You from me too.

**Comment by j-m-hoffmann** [comment]
rls 1.2.1 and 1.2.0 work though, problem appears from 1.2.2 onwards

**Comment by alextes** [comment]
Fixed in 1.2.5 .
